.github/workflows/ci_action_standalone.yml

```
@@ -25,12 +25,12 @@ jobs:
     - name: Configure CMake
       shell: bash
       working-directory: ${{runner.workspace}}/build
-      run: cmake $GITHUB_WORKSPACE/standalone -DCMAKE_BUILD_TYPE=$BUILD_TYPE
+      run: cmake $GITHUB_WORKSPACE/standalone -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_TESTING=ON
 
     - name: Build
       working-directory: ${{runner.workspace}}/build
       shell: bash
-      run: cmake --build . --config $BUILD_TYPE -DBUILD_TESTING=ON
+      run: cmake --build . --config $BUILD_TYPE
```

```suggestion
      run: cmake --build .
```
see https://stackoverflow.com/questions/24460486/cmake-build-type-is-not-being-used-in-cmakelists-txt/24470998#24470998 or https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project